test/samples/sample_split_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rspec"
+
+describe 'SplittableGroup' do
+  it 'runs one test' do
+    sleep(1)
```

Comment:
Are you running this locally? Would it make a huge difference to make this 5 or even 10, to hopefully reduce flapping in travis? Alternately, can we determine if we're in travis and bump it up to something 'huge' like 20?